playwright.config.ts

```diff
@@ -24,10 +26,36 @@ const getWebServers = () => {
     stderr: "pipe",
   });
 
+  if (process.env.CROSSPOST_TEST) {
```

Comment:
This environment variable is used to switch between "normal" playwright tests and crossposting tests (though that's really an implementation detail which is hidden away behind `yarn ...` commands)